Assets/MRTK/Providers/WindowsMixedReality/XRSDK/Controllers/WindowsMixedRealityXRSDKArticulatedHand.cs

```diff
@@ -162,8 +162,6 @@ private void UpdateHandData(InputDevice inputDevice)
         /// <returns>The current Unity finger bone converted into an MRTK joint.</returns>
         private TrackedHandJoint ConvertToTrackedHandJoint(HandFinger finger, int index)
         {
-            Profiler.BeginSample("[MRTK] WindowsMixedRealityXRSDKArticulatdHand.ConvertToTrackedHandJoint");
```

Comment:
I chose to remove this, since it'd involve a bit of a rewrite to update this method to be instrumentable. I can change this, but I'm not sure profiling a single switch statement will tell you much?

Reply:
I agree with the decision.